source/features/conversation-links-on-repo-lists.tsx

```diff
@@ -34,20 +34,48 @@ function addConversationLinks(repositoryLink: HTMLAnchorElement): void {
 	);
 }
 
-const selectors = [
-	'a[itemprop="name codeRepository"]', // `isProfileRepoList`
-	'.repo-list-item .f4 a', // `isGlobalSearchResults`
-] as const;
+function addSearchConversationLinks(repositoryLink: HTMLAnchorElement): void {
+	const repository = repositoryLink.closest('[data-testid="results-list"] > div')!;
+
+	// Place before the update date ·
+	$('ul > span:last-of-type', repository)!.before(
```

Comment:
You can use this format instead of a new constant and $()
```suggestion
	repositoryLink
     .closest(blah blah)
     .querySelector('ul > span:last-of-type', repository)!
     .before(
```